static/src/js/app/vuex/mutations.js

```diff
@@ -68,6 +68,9 @@ export default {
   },
   [CREATE_PERSONAL_VOCAB_ENTRY]: (state, data) => {
     state.personalVocabAdded = data.created;
+    if (state.personalVocabList.entries) {
+      state.personalVocabList.entries = [data.data, ...state.personalVocabList.entries];
```

Comment:
I can understand how this allows someone to see the new entry right away, even in a long list, but would it match the actual order? It might be confusing to refresh the screen and see that item disappear (in an unsorted table). On that note, would a data table with an existing sort or filter automatically refresh its pages and sort order (as that might hide the entry as well)?

Reply:
It does not respect any ordering. I agree that it could be confusing in the way that you've described, but the other implementations that I thought of had equally confusing UX. I think its a 6 in one, half dozen in another situation. This seemed like a fine enough solution until there is some actual feedback after folks have been using the new table.